android-agent/src/androidTest/kotlin/io/opentelemetry/android/BasicInstrumentationTest.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android
+
+import android.app.Application
+import android.os.Build.VERSION_CODES.LOLLIPOP
+import android.os.Build.VERSION_CODES.UPSIDE_DOWN_CAKE
+import androidx.test.core.app.ApplicationProvider.getApplicationContext
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.opentelemetry.android.config.OtelRumConfig
+import org.junit.Assert.assertNotNull
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+internal class BasicInstrumentationTest {
+    @Config(sdk = [LOLLIPOP, UPSIDE_DOWN_CAKE])
```

Comment:
(Maybe not possible) though it'd be nice if the latest version could be updated automatically as soon as a new one is available.

Reply:
There is the "current development" value also but I've never used it and aren't sure of what the behaviour is. I wouldn't mind keeping it at whatever our compileSdk version is at just to reduce unexpected flakiness. That said, but I'm opposed to trying it in case y'all prefer that.

Reply:
Could also use the `minSdk` and `maxSdk` params but then it'd run for all versions I think, if it's fast it's okay otherwise probably not.